doc/src/modules/evalf.rst

```diff
@@ -70,7 +70,7 @@ This shows digits 999,951 through 1,000,000 of pi:
 
 
 High-precision calculations can be slow. It is recommended (but entirely
-optional) to install gmpy (https://code.google.com/p/gmpy/), which will
+optional) to install gmpy (https://code.google.com/archive/p/gmpy/), which will
```

Comment:
This should better be github link to gmpy2 than archived link.

Reply:
I was also thinking the same. Would do it in the next commit.